collections/_posts/2022-02-14-chia-improvement-proposals.en.md

```diff
@@ -1,7 +1,7 @@
 ---
 lang: en
 layout: post
-title: "CHia Improvement Proposals"
+title: "Chia Improvement Proposals"

```

Comment:
This was intentional. Do we have consensus on changing it?

Reply:
WHOOPS!

My auto correct tagged that as a ??? and I just looked at it in an vacuum of the word all alone and took it as a fat finger. The fact it's the outline of an acronym never crossed my mind. I'll reverse it.